core/src/services/oss/core.rs

```diff
@@ -162,6 +163,10 @@ impl OssCore {
             req = req.header("x-oss-range-behavior", "standard");
         }
 
+        if let Some(if_none_match) = if_none_match {
+            req = req.header(http::header::IF_NONE_MATCH, if_none_match);
```

Comment:
We can use `http::header::IF_NONE_MATCH` directly, so that we can use `IF_NONE_MATCH` here for better reading.

Reply:
Sure! I made some change at https://github.com/apache/incubator-opendal/pull/1997/commits/0f67380897c5230efe8e9cfb1229faa36408fca0, is it the preferred style?

Reply:
Yes!